bot.py

```diff
@@ -31,4 +31,41 @@ async def create_repo(ctx, repoName): #!git create_repo repo1
 	repo = user.create_repo(repoName)
 	await ctx.send("repository "+repoName+ " created!")
 
+
+# display open and closed issues
+
+@bot.command()
+async def issues(ctx, repoName, state): # !git issues MLH-Fellowship/github-discord-bot open
+	repo = g.get_repo(repoName)
+	issues = repo.get_issues(state=state)
+	if(issues.totalCount == 0):
+		await ctx.send("There are no issues that match your query")
+	else: 
+		for i in issues:
+			await ctx.send('Issue Title: ' + i.title + '\nIssue Number: ' + str(i.number) +'\nIssue Link: https://github.com/' + repoName + '/issues/' + str(i.number))
```

Comment:
I'm a bit of a critic of `else` statements ( [you don't need them](https://medium.com/@jamousgeorges/dont-use-else-66ee163deac3) ) but wouldn't you have the same behavior if you just removed the `else` here? I mean, if there are no issues the for loop will not run anyways.

Also, put the response strings in a [long string](https://stackoverflow.com/questions/8577027/how-to-declare-a-long-string-in-python) on a separate variable and just send the variable here (looks a bit more organized)